README.md

```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/mesosphere/marathon.png?label=ready,gui&title=Ready)](https://waffle.io/mesosphere/marathon?label=gui)
```

Comment:
@mlunoe This is how it's currently done in [Marathon](https://github.com/mesosphere/marathon/blob/master/README.md) – the idea is that the Ready counter tells the users how many stories have been "assessed" and are thus going to be worked on next. The link sends you to the full Waflle board, in our case with the `gui` label filter applied.